src/index.html

```diff
@@ -4,13 +4,28 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="#" data-qa="big"
+    method="get" class="bigBar">
```

Comment:
Read the checklist: https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md#html-formatting

Reply:
Could you please, be more specific what is wrong here?

Reply:
Yes, every attribute should be in new line if at all there are more than 2 attributes in one tag
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/64101898/839d40f8-0896-4f5a-84e1-f2ff6cf61301)